src/components/TodoTask.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { TodoItem } from '../interfaces/todo'
+
+interface Props {
+    todoTask: TodoItem;
+};
+
+const TodoTask = ({ todoTask }: Props) => {
+    return (
+        <div className="content" key={todoTask.id}>
```

Comment:
Please remove the prop attribute from here.

Reply:
Do you mean remove "Props" on 8th line? If I remove it the "todoTask" will has 'any' type (error)

Reply:
No, on line 10 the <div> element still has the "key" prop. That needs to be removed from here and brought outside of this component to the parent component line 16.